Loop/Managers/CGMManager.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import LoopKit
+import LoopKitUI
```

Comment:
Is this needed?

Reply:
Yes, the `CGMManagerDescriptor` type was moved from `LoopKit` to `LoopKitUI`.